add new getting started flow (#1986)
urbit/urbit.org: content/getting-started/installing-applications.md

+++
title = "Installing apps"
tag = "around"
weight = 3
+++

> Installing urbit applications is very simple.  Just search for the developer, and download the application.  We'll walk you through it.

Author: ```suggestion
Installing Urbit applications is simple. Search for the developer and download the application directly from them.  We'll walk you through it.
```

### 1. Finding applications

> To get an application, you'll need to know its developer Urbit ID, *e.g.* `~tirrel` or `~dister-fabnev-hinmur`, or the full Urbit address, *e.g.* `~tirrel/studio`.

Author: ```suggestion
To isntall an application, you'll need to know the ship it is distributed from, which is often its developer's Urbit ID, *e.g.* `~paldev` or `~dister-fabnev-hinmur`. If you know the application, you can enter the full shortcode, e.g. `~paldev/pals`. 
```

> This information is available applications in the [Applications directory](/ecosystem?type=applications) in the Ecosystem section of urbit.org.

Author: ```suggestion
You can find a [public directory of applications](/ecosystem?type=applications) in the Ecosystem section of urbit.org.
```

> Urbit is still in its early stages and not all applications are listed on urbit.org, so you’ll need to join groups and browse around the network to discover the newest applications as they are being brought up. 

Author: ```suggestion
Not all applications are listed on urbit.org, so you’ll discover more by actively using Urbit and meeting people.
```

> However, we can get started a simple app from Urbit developer [`~tirrel`](/ids/~tirrel).

Author: Studio is broken at present after the Groups migration. This is not a good one to recommend. Stick with something battle-tested and actively updated like `pals`.
